builder/builder_test.go

```diff
@@ -80,7 +80,7 @@ func TestBuild(t *testing.T) {
 
 	for description, test := range tests {
 		t.Run(description, func(t *testing.T) {
-			inclusionListTxs := testapp.ToTxs(t, test.inclusionList)
+			inclusionListTxs := append([][]byte{testutils.GenerateBlock(t).Txs[0]}, testapp.ToTxs(t, test.inclusionList)...)
```

Comment:
Potential new issue:

There's a mismatch between this test setup and current monomer behaviour. The appchain transactions here are bundled as deposit txs, rather than sourced from the builder's mempool.

As far as I know, we are missing some plumbing to permit arbitrary appchain TXs through the deposit tx workflow. These tests do demonstrate that some pieces of that workflow are present.

Reply:
In verifier mode, the `InjectedTransactions` can include more than just deposit txs. Am I missing something?

Reply:
Makes sense, yup.